scripts/kvm-test.py

```diff
@@ -193,6 +190,16 @@ def load_config(self):
                     help='boot test image')
 
 
+cc_group = parser.add_mutually_exclusive_group()
+cc_group.add_argument('-f', '--autoinstall-file', action='store',
+                      type=argparse.FileType(),
+                      help='load autoinstall from file and pass "autoinstall"'
+                           ' to the kernel command line')
+cc_group.add_argument('--cloud-config-file', action='store',
```

Comment:
What do you think about just calling this arg `--cloud-config`?

Reply:
Yeah, I don't mind. I tried to mimic `--autoinstall-file` for consistency but if we are getting rid of it anyway ...